tests/ui/attributes/z-crate-attr/injection.rs

```diff
@@ -0,0 +1,3 @@
+//@ compile-flags: '-Zcrate-attr=feature(yeet_expr)]fn main(){}#[inline'
```

Comment:
What about `-Zcrate-attr=feature(yeet_expr)]#![allow(warnings)`? aka when you have two attributes back to back rather than an item after an attribute.

Reply:
that is also rejected, but i've added a test to verify.